kubejs/server_scripts/mods/ae2/gregifyAE2.js

```diff
@@ -0,0 +1,1897 @@
+let gregifyAE2 = (/** @type {Internal.RecipesEventJS} */ event) => {
+    //Formation & Annihilation Cores
+        event.remove(
+            [
+                {
+                    id: 'ae2:materials/formationcore'
+                },
+                {
+                    id: 'ae2:materials/annihilationcore'
+                }
+            ]
+        )
+
+        event.recipes.gtceu.assembler('gregitas:formation_core')
+            .itemInputs(
+                'gtceu:certus_quartz_gem',
+                'ae2:fluix_dust',
+                'ae2:logic_processor'
+            )
+            .inputFluids(
+                Fluid.of(
+                    'gtceu:red_alloy',
+                    144
+                )

```

Comment:
could you just do `'gtceu:red_alloy 144'`? it'd be way more compact
or at least put this on 1 line